src/transforms/encryption.js

```diff
@@ -0,0 +1,149 @@
+            if (length > 2)
+                ba.push((word >>> 8) & xFF);
+            if (length > 3)
+                ba.push(word & xFF);
+
+            return ba;
+        }
+
+        wordArrayToByteArray(wordArray, length) {
+            if (wordArray.hasOwnProperty("sigBytes") && wordArray.hasOwnProperty("words")) {
+                length = wordArray.sigBytes;
+                wordArray = wordArray.words;
+            }
+
+            var result = [],
+                bytes,
+                i = 0;
+            while (length > 0) {
+                bytes = this.wordToByteArray(wordArray[i], Math.min(4, length));
+                length -= bytes.length;
+                result.push(bytes);
+                i++;
+            }
+            return [].concat.apply([], result);
+        }
+
+
+        _transform(chunk, enc, cb) {
+            try {
+                //console.log("enc:" + enc);
+                //console.log("c-in", chunk)
+
+                let encrypted = CryptoJS.AES.encrypt(
+                    CryptoJS.enc.Hex.parse(chunk.toString('hex')),
+                    CryptoJS.enc.Hex.parse(this.key.toString('hex')), {
+                        iv: CryptoJS.enc.Hex.parse(this.iv.toString('hex')),
+                        mode: CryptoJS.mode.CFBb,
+                        segmentSize: 8,
+                        padding: CryptoJS.pad.NoPadding
+                    }
+                )
+
+                //console.log("ct:", encrypted.ciphertext)
+                let cout = Buffer.from(this.wordArrayToByteArray(encrypted.ciphertext));
+                //console.log("c-out", cout)
+                this.iv = cout.slice(cout.length - 16, cout.length);
+
+                cb(null, cout);
+            } catch (e) {
+                cb(e); //This can cause "Error [ERR_MULTIPLE_CALLBACK]: Callback called multiple times" when an error was thrown while calling cb in the try block.
+            }
+        }
+    }
+
+    class Decipher extends Cipher {
+        _transform(chunk, enc, cb) {
+            try {
+                //console.log("enc:" + enc);
+                //console.log("dc-in", chunk)
+
+                let inp = CryptoJS.enc.Hex.parse(chunk.toString('hex')); //hex and binary
+                //console.log("dc-in2", inp);
+
+                let decrypted = CryptoJS.AES.decrypt( //
+                    { ciphertext: inp },
+                    CryptoJS.enc.Hex.parse(this.key.toString('hex')), //
+                    {
+                        iv: CryptoJS.enc.Hex.parse(this.iv.toString('hex')),
+                        mode: CryptoJS.mode.CFBb,
+                        segmentSize: 8,
+                        padding: CryptoJS.pad.NoPadding
+                    }
+                );
+
+                let resbuf = Buffer.from(this.wordArrayToByteArray(decrypted));
+                //console.log("dc-out", resbuf)
+                this.iv = chunk.slice(chunk.length - 16, chunk.length);
```

Comment:
Not convinced this stream can be parsed tcp packet by tcp packet. I encourage you look this up and maybe look at the code of the crypto internal package for that